panther_hardware_interfaces/src/panther_imu.cpp

```diff
@@ -35,8 +35,6 @@ namespace panther_hardware_interfaces
 
 CallbackReturn PantherImuSensor::on_init(const hardware_interface::HardwareInfo & hardware_info)
 {
-  RCLCPP_INFO(logger_, "Initializing Panther IMU");
```

Comment:
Sam as for panther_system

Reply:
Please refer to [comment](https://github.com/husarion/panther_ros/pull/320#discussion_r1627460829)